publiq-ui/pub-alert.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <b-alert :variant="variant" :show="show" dismissible>
+    <slot />
+  </b-alert>
+</template>
+
+<script>
+  export const AlertVariant = {
+    DANGER: 'danger',
+    INFO: 'info',
+    SUCCESS: 'success',
+  };
+
+  export default {
+    props: {
+      variant: {
+        type: String,
+        default: AlertVariant.INFO,
+      },
+      show: {
```

Comment:
Nit: In the modal PR we use `visible`. I know it's just copied from the Bootstrap prop name, but on our own components we could of course name it whatever we like and use `visible` or `show` consistently. Not blocking though, I leave it up to you to decide what you think is best / worth it.